main/states/busy_state.py

```diff
@@ -50,7 +50,10 @@ def on_enter(self, payload=None):
                     os.system('play ' + audio_url[6:])  # nosec #pylint-disable type: ignore
 
             if 'volume' in reply.keys():
-                subprocess.call(["amixer", "-D", "pulse", "sset", "Master", str(reply['volume'])])  # nosec #pylint-disable type: ignore
+                m = alsaaudio.Mixer()
+                m.setvolume(int(reply['volume']))
+                os.system('play {0} &'.format(
+                self.components.config['detection_bell_sound']))
```

Comment:
`detection_bell_sound` - does this play the bell sound when the volume has reached the desired level?

Reply:
Yes